src/main.py

```diff
@@ -7,12 +7,35 @@
 import random
 from validate import validate, SUPPORTED_YEARS, DEFAULT_YEAR
 
-app = Flask(__name__)
+# Set WOFF and WOFF2 caching to return 1 year as they should never change
+# Note this requires similar set up in app.yaml for Google App Engine
+class MyFlask(Flask):
```

Comment:
We could maybe move the `MyFlask` class into a separate file and import it here? Is there no way to do the `app.config['SEND_FILE_MAX_AGE_DEFAULT'] = 10800` within the custom flask class, rather than external to it? (Just asking, I'm not sure.) 

Otherwise, it looks like a good change.

Reply:
Not sure of benefit of externalising MyFlask, when it's only there to override one function? Would external files for such a simple use case not just cause more readability issues than help?

To be honest I'd rather not have this MyFlask class at all, but was the only way I could see to override the `get_send_file_max_age` function to allow per extension defaults. Presume there's no other way but to instantiate another class (never used Python before tbh!)?

And, it's maybe even overkill to have this as it's not strictly necessary, as could just live with the 3 hours for fonts too in dev, but found it confusing already to have different values in production (app.yaml) and dev (main.py) so decided to add this here to avoid that confusion (somewhat!) in future.

I tried again to set that config on the custom class but couldn't. Open to suggestions here but gain not sure what gains there are from doing that? Isn't the current way of doing it with `app.config` similar to how CSP is set on the `app` instance in the following lines?

```python
app.config['SEND_FILE_MAX_AGE_DEFAULT'] = 10800
Talisman(app,
         content_security_policy=csp,
         content_security_policy_nonce_in=['script-src'])
```

Reply:
Ok cool. I think consistency between dev and prod is important, so we should do it.

The rest was just a thought. With any MVC-styled app (regardless of language), I just try and keep things as tidy as possible. Single file, single responsibility as far as is reasonable (but I agree there can be diminishing value for such small things). I would personally prefer to separate any 'bootstrapping' concerns such as the CSP from the controller logic, but again - it may be overkill in this case.

Otherwise, it's a good change, feel free to resolve and merge. :)

Reply:
OK will leave as is for now and merge. You can do separate PR to tidy up both this, and CSP (and anything else) if you want.